django_lightweight_queue/worker.py

```diff
@@ -155,12 +159,22 @@ def process(self, backend):
         return True
 
     def tell_master(self, timeout, sigkill_on_stop):
-        self.back_channel.put((
-            self.queue,
-            self.worker_num,
-            timeout,
-            sigkill_on_stop,
-        ))
+        if sigkill_on_stop:
+            # SIGUSR2 can be taken to just cause the process to die
+            # immediately.
```

Comment:
Might be worth mentioning that is the default action that's taken.

Reply:
5fec5658bb1146af16f39992536b1341e394545d